lib/resources/r4/organization.yaml

```diff
@@ -104,7 +98,6 @@ fields:
 
   - name: name
     required: 'Yes'
```

Comment:
double check

Reply:
Based on the note and verifiers I believe this is required.